eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -79,11 +79,14 @@ stages:
             - ${{ replacement }}
 
           - AZURE_TEST.*=.*/
+          - ${{ if eq(parameters.SDKType, 'data') }}:
+            - JavaTestVersion=(.*1)\.17(.*)/$1.11$2
```

Comment:
Looks weird but is needed to ensure there is a non-digit character after the `$1` regex capture group usage

Reply:
So data should be 1.11, and client 1.17?

Reply:
Yes, data will be 1.11 and client 1.17 (except in cases where there are known issues with using 1.17)

Reply:
Do we not need to test any 1.11 any longer? 

For the data (aka. track 1) are we doing this only because of cost currently with the intention to support 17 later or do we plan to never support it?

Reply:
I'd throw one java 11 (or better java 9) job into matrix. Given that core is MRJAR and has code that becomes active on Java9+.

Reply:
I'd throw one java 11 (or better java 9) job into matrix. Given that core is MRJAR and has code that becomes active on Java9+.

Reply:
It'll need to be Java 11 for now as what DevOps does behind the scenes is sets the JDK to Java 11 and Java 9 won't be installed as a default.